Project.swift

```diff
@@ -86,9 +87,12 @@ let project = Project(name: "kDrive",
                                  platform: .iOS,
                                  product: .unitTests,
                                  bundleId: "com.infomaniak.drive.tests",
-                                 deploymentTarget: .iOS(targetVersion: "13.0", devices: [.iphone]),
+                                 deploymentTarget: .iOS(targetVersion: "13.4", devices: [.iphone]),
                                  infoPlist: .file(path: "kDriveTests/Tests.plist"),
                                  sources: "kDriveTests/**",
+                                 resources: [
+                                     "kDriveTests/**/*.jpg"
```

Comment:
I used tuist to import the ressourses I need in the test target. cc @PhilippeWeidmann